HealthcareSystem/GraphicalEditor/Models/MapObjectRelated/MapObjectDoor.cs

```diff
@@ -0,0 +1,114 @@
+﻿using GraphicalEditor.Enumerations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Controls;
+using System.Windows.Media;
+using System.Windows.Shapes;
+
+namespace GraphicalEditor.Models.MapObjectRelated
+{
+    public class MapObjectDoor
+    {
+        private Rectangle _rectangle;
+        private MapObjectMetrics _mapObjectMetrics;
+        private MapObjectDoorOrientation _mapObjectDoorOrientation;
+        private double _XShift;
+        private double _YShift;
+
+        private readonly double DOOR_WIDTH = 45;
+        private readonly double DOOR_HEIGHT = 10;
+
```

Comment:
Could you make only one file with all constants? For an easy change in the future.

Reply:
Ok, no problem.